ci/uninstall-thin-edge_io/uninstall-thin-edge_io.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
+
+set -e
+
+usage() {
+    cat <<EOF
+USAGE:
+    delete-thin-edge_io [COMMAND]
+    
+COMMANDS:
+    remove     Uninstall thin-edge.io with keeping configuration files
+    purge      Uninstall thin-edge.io and also remove configuration files  
+
+EOF
+}
+
+stop_a_service_if_running() {
+    status=$(sudo systemctl is-active "$1") && returncode=$? || returncode=$?
+    if [ "$status" = "active" ]; then
+        sudo systemctl stop "$1"
+    fi
+}
+
+stop_extension_services() {
+    stop_a_service_if_running "tedge-watchdog.service"
+    stop_a_service_if_running "tedge-mapper-collectd.service"
+    stop_a_service_if_running "c8y-log-plugin.service"
+    stop_a_service_if_running "c8y-configuration-plugin.service"
+}
+
+remove_or_purge_package_if_exists() {
+    status=$(dpkg -s "$2" | grep -w installed) && returncode=$? || returncode=$?
```

Comment:
```suggestion
    local status
    status=$(dpkg -s "$2" | grep -w installed) && returncode=$? || returncode=$?
```

Same question about the `returncode` variable: You're not using this anywhere...

Reply:
Catching the `returncode` of the command to avoid the exiting of the script, if `returncode` is non-zero.

Reply:
Ah, ok.